packages/perspective/test/js/leaks.spec.js

```diff
@@ -71,4 +97,77 @@ describe("leaks", function() {
             table.delete();
         });
     });
+
+    describe("expression columns", function() {
+        it("0 sided does not leak", async () => {
+            const table = await perspective.table({
+                a: [1, 2, 3, 4],
+                b: [1.5, 2.5, 3.5, 4.5],
+                c: ["a", "b", "c", "d"],
+                d: [new Date(), new Date(), new Date(), new Date()]
+            });
+
+            const expressions = generate_expressions();
+
+            await leak_test(async () => {
+                const view = await table.view({
+                    expressions: [expressions[Math.floor(Math.random() * expressions.length)]]
+                });
+                const expression_schema = await view.expression_schema();
+                expect(Object.keys(expression_schema).length).toEqual(1);
+                await view.delete();
+            });
+
+            await table.delete();
+        });
+
+        it("1 sided does not leak", async () => {
+            const table = await perspective.table({
+                a: [1, 2, 3, 4],
+                b: [1.5, 2.5, 3.5, 4.5],
+                c: ["a", "b", "c", "d"],
+                d: [new Date(), new Date(), new Date(), new Date()]
+            });
+
+            const columns = ["a", "b", "c", "d"];
+            const expressions = generate_expressions();
+
+            await leak_test(async () => {
+                const view = await table.view({
+                    row_pivots: [columns[Math.floor(Math.random() * columns.length)]],
+                    expressions: [expressions[Math.floor(Math.random() * expressions.length)]]
+                });
+                const expression_schema = await view.expression_schema();
+                expect(Object.keys(expression_schema).length).toEqual(1);
+                await view.delete();
+            }, 3000);
```

Comment:
These looks correct on-the-face, but why 3,000?  In theory, these test should run identically with 1 or 1,000,000 - iteration checks are just there for offensively slow operations.